SECURITY.md

```diff
@@ -0,0 +1,72 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+The OPI security policy aims to efficiently handle security vulnerabilities
+while minimizing public disclosure until a fix is available.
+
+Reporters are requested to submit security issues privately via the
+[GitHub advisory](https://github.com/opiproject/opi/security/advisories/new)
+system.
+
+Reports will be acknowledged within 5 calendar days. OPI maintainers will
+respond to a report within a maximum of calendar 10 days, indicating the
+next steps in handling the submission.
+
+## Reporting a Vulnerability in 3rd Party Modules
+
+Security issues in third-party modules should be reported according to their
+own security policies or directly to their respective maintainers if no such
+security policy is in place for the respective module.
+
+The focus of this security policy is on reporting OPI security vulnerabilities.
```

Comment:
@jlwhite I am against including "future" work information here - just what is there currently. At any time we can extend it with a more detailed policy on handling 3rd party issues. I adjusted by restating the focus of the current proposal.